core/server/web/api/v2/admin/uploadHelper.js

```diff
@@ -0,0 +1,27 @@
+const shared = require('../../../shared');
+const config = require('../../../../config');
+
+const multer = {
+    clear: config.get('upload-clear') || true,
```

Comment:
```suggestion
    clear: config.get('uploadClear') || true,
```

Config files are following camelCase naming, would be good to keep the convention intact :+1: So something like 'uploadClear' would be better